CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
```

Comment:
```suggestion
### Updated
```

Reply:
The keepachangelog format doesn’t support this headline. LGTM as is

Reply:
Oh, we have some mistakes in CHANGELOG.md, then. 😳

Reply:
What is done is done. Let’s keep in mind next times.